packages/field-base/src/label-controller.js

```diff
@@ -0,0 +1,208 @@
+/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { SlotController } from '@vaadin/component-base/src/slot-controller.js';
+
+/**
+ * A controller to manage the label element.
+ */
+export class LabelController extends SlotController {
+  constructor(host, labelId) {
+    super(
+      host,
+      'label',
+      () => document.createElement('label'),
+      (host, node) => {
+        this.label = host.label;
```

Comment:
This shouldn't probably be needed since there is a `setLabel` method.

Reply:
Done.